packages/mosaic/popover/_popover-theme.scss

```diff
@@ -95,6 +95,10 @@
 }
 
 @mixin mc-popover-typography($config) {
+    .mc-popover__container {
```

Comment:
`mc-popover__container` has 3 nested items (`mc-popover__header, mc-popover__content, mc-popover__footer`) and they have own typography:
![image](https://user-images.githubusercontent.com/9027254/70418383-4c7cd780-1a74-11ea-8593-7a38ca1f312b.png)

I think that typography for `mc-popover__container` is redundant and you should delete this.